src/Symfony/Component/Serializer/Mapping/ClassMetadataInterface.php

```diff
@@ -41,7 +41,7 @@ public function addAttributeMetadata(AttributeMetadataInterface $attributeMetada
     /**
      * Gets the list of {@link AttributeMetadataInterface}.
      *
-     * @return AttributeMetadataInterface[]
+     * @return array An array of {@see AttributeMetadataInterface} instances.
```

Comment:
Do the keys convey a special meaning? Or what's the advantage over using `AttributeMetadataInterface[]`?